lessons/python_packages/source-lesson.md

```diff
@@ -0,0 +1,217 @@
+# Creating Packages in Python
```

Comment:
I'm going to assume this is the original lesson by Bill and you aren't going to present it, so I won't post any comments about it, but I can if you want it.